src/components/FeatureViews/Registration/RegistrationCourseEnrollments.js

```diff
@@ -91,24 +98,44 @@ const RegistrationCourseEnrollments = ({courseID}) => {
             }), {}));
     }, [course.roster]);
 
+    useEffect(()=>{
+        api.fetchAllSessions({
+            "config": {
+                "params": {
+                    "time_frame": "month",
+                    "time_shift": 0,
+                },
+            },
+            "id": "",
+        });
+    },[api]);
```

Comment:
You can utilize the `useClassSessionsInPeriod/useTutoringSessionsInPeriod` hook in place of this, in hooks.js, or modify/create your own to fetch both at once (refer to code for how to do this).